### PR TITLE
try to fix the build

### PR DESCRIPTION
**Pull request by epistemancer**
* fix the circleCI config so that it uses the correct GPG key to sign code in the deploy step
* update versions of some dependencies (may not have been necessary but doesn't hurt)
* fix getter/setter for "Address Importer" field, and add for "Invoiced Charges" field, on CustomsDeclaration

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -54,6 +54,8 @@ jobs:
             # fallback to using the latest cache if no exact match is found
               - v2-dependencies-
         - run: printf "scm.url=scm:git:https://github.com/goshippo/shippo-java-client.git\nscm.tag=release" > release.properties
+        # load the GPG key again as this is a new shell
+        - run: printf -- "$gpg_secret" | gpg --batch --yes --passphrase=$gpg_password --pinentry-mode loopback --import
```
Comment (epistemancer) [inline .circleci/config.yml:58]: this was the root of the issue with the release job.  The key was imported in the build step, but not in the deploy step, which runs in a different shell; so the deploy step was defaulting to some old key from the VM image, which has expired.

File: src/main/java/com/shippo/model/CustomsDeclaration.java [hunk 2]
```diff
@@ -248,12 +250,12 @@ public void setMetadata(Object metadata) {
 		this.metadata = metadata;
 	}
 
-	public Object getAdressImport() {
+	public Object getAddressImporter() {
 		return metadata;
```
Comment (sam-allen-shippo) [inline src/main/java/com/shippo/model/CustomsDeclaration.java:254]: should this still be returning `metadata`?

Comment (epistemancer) [reply src/main/java/com/shippo/model/CustomsDeclaration.java:254]: it should not.  Bad copypasta, thanks for catching.

File: src/main/java/com/shippo/model/CustomsDeclaration.java [hunk 3]
```diff
@@ -271,4 +273,12 @@ public Object getB13aNumber() {
 	public void setB13aNumber(Object b13aNumber) {
 		this.b13aNumber = b13aNumber;
 	}
+
+	public Object getInvoicedCharges() {
+        return invoicedCharges;
```
Comment (sam-allen-shippo) [inline src/main/java/com/shippo/model/CustomsDeclaration.java:278]: is this another undocumented field?

Comment (epistemancer) [reply src/main/java/com/shippo/model/CustomsDeclaration.java:278]: it's semi-documented... it's documented for the CustomsDeclaration object, but not for the POST payload in particular.  But it's a valid POST attribute.

**Review by sam-allen-shippo — APPROVED** [review]

**Pull request merged**